internal_ws/ykbh/src/lib.rs

```diff
@@ -194,9 +194,9 @@ impl SIRInterpreter {
     /// Run the SIR interpreter after it has been initialised by a guard failure. Since we start in
     /// the block where the guard failed, we immediately skip to the terminator and interpret it to
     /// see which block we need to start interpretation in.
-    pub unsafe fn interpret(&mut self, tio: *mut u8) {
-        // Set the pointer to the trace inputs.
-        self.set_trace_inputs(tio);
+    pub unsafe fn interpret(&mut self, ctx: *mut u8) {
+        // Set the pointer to the trace ctx.
```

Comment:
There are still some references to "trace context" in comments.

[cloned you branch before grepping this time!]

Reply:
Fixed in https://github.com/softdevteam/yk/pull/222/commits/2a8bd6d9b15d4faba4147363e7b498479d2c66f7.